plugins/wake/wake.plugin.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# oh-my-bash.module: wake-on-lan wrapper + autocompletion
+# wake.plugin.sh
+# Author: hoek from 0ut3r.space
+# Based on oh-my-zsh wake plugin @ commit 1d9eacb34f59f3bf82a9de0d7b474cb4c501e3fd
```

Comment:
I checked the upstream OMZ, but the name of the upstream plugin isn't `wake` but `wakeonlan`.